backend/settings.py

```diff
@@ -1,4 +1,12 @@
-BEER_PAGE_LIMIT = 10
-ORDERS_PAGE_LIMIT = 20
-STOCK_PAGE_LIMIT = 20
-USER_PAGE_LIMIT = 20
+import os
+
+BEER_PAGE_LIMIT = int(os.getenv("BEER_PAGE_LIMIT", "20"))
+ORDERS_PAGE_LIMIT = int(os.getenv("BEER_PAGE_LIMIT", "20"))
+STOCK_PAGE_LIMIT = int(os.getenv("BEER_PAGE_LIMIT", "20"))
+USER_PAGE_LIMIT = int(os.getenv("BEER_PAGE_LIMIT", "20"))
+
+DATABASE = "beerpy"
+PGHOST = "localhost"
+PORT = "5432"
+USER = "aviv"
+PASSWORD = "AsPsql23"
```

Comment:
1. we don't commit passwords
2. change the settings to be read from the environment